server/src/test/java/com/hedera/block/server/persistence/storage/read/BlockAsDirReaderTest.java

```diff
@@ -52,23 +51,19 @@
 import org.junit.jupiter.api.AfterEach;
 import org.junit.jupiter.api.BeforeEach;
 import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.io.TempDir;
 
 public class BlockAsDirReaderTest {
 
     private final Logger LOGGER = System.getLogger(getClass().getName());
 
-    private static final String TEMP_DIR = "block-node-unit-test-dir";
-
-    private Path testPath;
+    @TempDir private Path testPath;
```

Comment:
We do not need cleanup in the `@AfterEach` method, by default junit will attempt to recursively delete the `@TempDir` and if it does not succeed it should throw an exception and the test should fail (this should be based on documentation). Consider to take a look if we could remove this teardown logic and if indeed everything cleans up without it by default.

Reply:
✅  Done!